### ISSUE TITLE
Closing Costs link in the Rate Checker page lands on a different page 

### ISSUE THREAD

**Issue by OrlandoSoto** [cfpb/owning-a-home#532]
On the build server "owning-a-home/check-rates/" 
The "closing costs" link seems to be pointing to: 
http://www.consumerfinance.gov/askcfpb/139/what-are-closing-costs.html

But instead when I click on it, the browser lands in:
http://www.consumerfinance.gov/askcfpb/1845/what-fees-or-charges-are-paid-closing-and-who-pays-them.html

So the browser test fails the assertion as it correctly detects that we did not land in the URL we expected

A couple things we can do:
- Change the expected URL browser test 
- Change the href attribute in the closing costs link

![closingcosts](https://cloud.githubusercontent.com/assets/7442823/8913629/14c66dd4-3468-11e5-95ac-2b88d7262078.png)

**Comment by mthibos** [comment]
@OrlandoSoto  thanks for noticing!

I'm double checking, but that's probably because we have discontinued the original link and consolidated with the link where it's actually landing.  If so, we should just change the link in the code (and test).

**Comment by OrlandoSoto** [comment]
thanks @mthibos !

**Comment by mthibos** [comment]
Yep, that's what's going on.  The redirect is permanent.

@stephanieosan can you make that change in the code on build?

**Comment by stephanieosan** [comment]
Yes, I can update the code and the test. @mthibos, this is the correct link, right?

http://www.consumerfinance.gov/askcfpb/1845/what-fees-or-charges-are-paid-closing-and-who-pays-them.html

**Comment by mthibos** [comment]
yep!

**Comment by OrlandoSoto** [comment]
@stephanieosan I updated the test in https://github.com/cfpb/owning-a-home/pull/534

**Comment by stephanieosan** [comment]
Oh, nice! I will do nothing then :smile:

**Comment by OrlandoSoto** [comment]
Browser test fixed in https://github.com/OrlandoSoto/owning-a-home/commit/cd2b9ead5e5ef1ab216119f195eff96556d2f7b1
